actions/bind

```diff
@@ -173,6 +179,37 @@ def set_forwarders(forwarders):
     conf_file.close()
 
 
+def set_adblock(choice):
+    """Enable or disable ad blocking."""
+    data = [line.strip() for line in open(CONFIG_FILE, 'r')]
+    if choice == 'true':
+        # serve a single pixel gif for blocked domains
+        action_utils.webserver_enable('adblock-plinth')
+
+        # fetch ad server list
```

Comment:
Instead of using a single list, how about using a list aggregated from a lot of sources, like in this Github repo. https://github.com/StevenBlack/hosts

The list at pgl.yoyo.org has 2324 entries while the aggregated one has close to 40,000 entries. 
See the sample list here. https://raw.githubusercontent.com/StevenBlack/hosts/master/hosts